src/main/scala/org/constellation/API.scala

```diff
@@ -401,15 +428,15 @@ class API(udpAddress: InetSocketAddress)(implicit system: ActorSystem,
       path("health") {
         complete(StatusCodes.OK)
       } ~
-        path("favicon.ico") {
-          getFromResource("favicon.ico")
+        path("ui/img/favicon.ico") {
```

Comment:
but the browser checks the path /favicon.ico for favicons. By moving it it won't work anymore.

Reply:
Oh woops will fix

Reply:
I had this in the rendered HTML as         link(rel := "icon", href := "/img/favicon.ico"),

but I'll make it the standard again